src/MySqlConnector/Core/SchemaProvider.cs

```diff
@@ -33,6 +34,35 @@ public DataTable GetSchema(string collectionName)
 			return dataTable;
 		}
 
+		public DataTable GetSchema(string collectionName, string[] restrictions)
```

Comment:
I don't think introducing a new method should be necessary; `string[] restrictions` could be added as a new parameter to the existing method.

Reply:
Agreed. Can I make the default value `null` then?

Reply:
Yes, that sounds fine.